### ISSUE TITLE
Reply to comments in contacts and groups

### ISSUE THREAD

**Issue by micahmills** [DiscipleTools/disciple-tools-theme#987]
It might be useful to create a mechanism to reply to a previous comment possibly like a thread in Slack or a Reply in WhatsApp. This would allow someone to respond to a previous comment and the comment and reply would be linked together.

This could possibly be implemented at the same time as #537

**Comment by corsacca** [comment]
Work on this issue has been started here: https://github.com/DiscipleTools/disciple-tools-theme/pull/1187
I'm hesitant about new comments being added at the under existing comments instead of at the top of the lisnt. I think users could easily miss that a new comment was added.
@micahmills any thoughts on this?

**Comment by micahmills** [comment]
Yes this could lead to some difficult UI things. I think we could put a pause on this. It might be something that we try again in the future, but it will need some design and UI brainstorming.
